packages/preview/minimalbc/0.0.1/lib.typ

```diff
@@ -0,0 +1,88 @@
+#let companytext(size, body, style: "normal") = text(
+  size: size,
+  style: style,
+  weight: "medium",
+)[#body]
+#let nametext(size, body, style: "normal") = text(
+  size: size,
+  style: style,
+  weight: "medium",
+)[#body]
+#let roletext(size, body, style: "normal") = text(
+  size: size,
+  style: style,
+  weight: "light",
+)[#body]
+
+#let minimalbc(
+  company_name: "Company Name",
+  name: "First and Last Name",
+  role: none,
+  telephone_number: "+00 000 0000000",
+  email_address: "me@me.com",
+  website: "example.com",
+  company_logo: none,
+  bc_background: none,
+  bg_color: "f2f2f2",
+  geo_size: "eu",
+  flip:true,
+  body
+) = {
+    set text(font: "Lucida Sans")
+    set text(9pt)
+    let pz = geo_size+"-business-card"
+    set page(paper:pz, flipped:flip, fill: rgb(bg_color))
+
+    grid(columns:(100%),
+        rows:(50%,10%,35%),
+        gutter:0.5mm,
+        grid(
+    columns: (60%, 40%),
+    rows: (40%, 15%, 20%,  20% ),
+    gutter: 1mm,
+    companytext(14pt,company_name),
+    grid.cell(rowspan:2, image(company_logo), align:center),
```

Comment:
File path in packages are resolved relatively to the package: this means that the only value that will work for the `company_logo` are the images shipped with the package. The workaround is to let the user call `image` themselves and pass that as an argument to your package.

Reply:
I'm not entirely certain about this, but it's expected that users save the company logo using the specific filename 'company_logo.png'. I plan to elaborate on this and provide further clarification in the README. Currently, my time to devote to this project is limited. However, please feel free to make additions and modifications as you see fit.

Reply:
Even if they save their own logo under a file with the same name, this won't work as expected. Passing the `image` element directly instead of just a path is the only way to make it work at the moment.

Reply:
Done